templates/index.html

```diff
@@ -67,6 +67,8 @@
   <div class="hidden" id="host-skeleton"><a class="GitHubCommitURL" href=""></a> <span class="hidden"> (<a class="GitHubDiffURL" href="">diff</a>)</span></div>
 
   <script type="text/javascript">
+  GITHUB_TOKEN = "{{.GithubToken}}";
```

Comment:
It would be better if we could avoid sending server-side long-living credentials back to client every time.
Do you have any idea to achieve that?